webapp/packages/core-sdk/src/queries/fragments/DriverMainPropertyInfo.gql

```diff
@@ -0,0 +1,14 @@
+fragment DriverMainPropertyInfo on ObjectPropertyInfo {
+  id
+  displayName
+  description
+  category
+  dataType
+  defaultValue
+  validValues
+  length
+  features
+  order
+  supportedConfigurationTypes
+  required
+}
```

Comment:
we have a special fragment that used with `ObjectPropertyInfoForm` it's preferable to use it instead when we want to pass data to this control (if we don't have it then it's time to create one and use it)